docs/List.md

```diff
@@ -1174,6 +1174,26 @@ const ProductList = () => (
 )
 ```
 
+## Enabling Data Fetching Conditionally
```

Comment:
I'm not sure about where to put this documentation as it's actually `useGetList` that we configure.

Reply:
I think it's fine here 👍